### DISCUSSION TITLE
OPT-301-ReviewBody

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#692] [Q&A] [answered]
Hello Labs,

There is a field: "OPT-301-ReviewBody", and we wanted to use this for Review body organisation, but we cannot.
This field is forbidden on all notifications. 
Can you tell me the purpose of "OPT-301-ReviewBody"?

We use SDK 1.7.1 but there is no change in the newer SDK versions too.

Thanks in advance
Anna

**Comment by YvesJo** [comment, marked as answer]
The field "OPT-301-ReviewBody" is exclusively for E5 as part of BG-714 associated roles, the one you're looking for is indeed "OPT-301-Lot-ReviewOrg".

**Comment by AnnaKSZ** [comment]
Thanks!
Anna
